react-components/src/components/MyAccount/myAccount.js

```diff
@@ -27,7 +27,7 @@ import { useConfigContext } from '../../context/ConfigContext';
 const MyAccount = props => {
     const { showMenu, showChangePassword, showAccountInformation } = props;
     const [{ currentUser, inProgress }, { signOut }] = useUserContext();
-    const { pagePaths } = useConfigContext();
```

Comment:
Please remove the unused variable `storeView` here.

Reply:
Removed!